src/io/iomap.cpp

```diff
@@ -57,7 +57,7 @@ void IOMap::loadMap(Map* map, const Position &pos) {
 	uint32_t majorVersionItems = stream.getU32();
 	stream.getU32(); // minorVersionItems
 
-	if (version > 2) {
+	if (version > 5) {
```

Comment:
I was thinking here and, we can just send a "warning" if the map version is more than 5, it will possibly still open normally, as version 5+ is just to define that it will work with the assets